### PR TITLE
android acs 2.0.4 release notes

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/adobe-campaign-standard/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## May 25, 2023
+
+### Android Campaign Standard 2.0.4
+
+* Update the AEPCore dependency to version 2.2.0 to resolve a WebView transparency issue and fix message frequency rules for ACS in-app messages.
```
Comment (calebk1) [inline src/pages/documentation/adobe-campaign-standard/release-notes.md:7]: ACS = Adobe Campaign Standard?

I think we should probably make it "... for Campaign Standard in-app messages." instead.

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 25, 2023
+
+### Android Campaign Standard 2.0.4
+
+* Update the AEPCore dependency to version 2.2.0 to resolve a WebView transparency issue and fix message frequency rules for ACS in-app messages.
```
Comment (calebk1) [inline src/pages/documentation/release-notes/index.md:11]: Same comment as above.

**Review by calebk1 — COMMENTED** [review]
Small comments!

File: src/pages/documentation/adobe-campaign-standard/release-notes.md (hunk 1 above)
Comment (rymorale) [reply src/pages/documentation/adobe-campaign-standard/release-notes.md:7]: you are correct, it is Adobe Campaign Standard. good point, will fix it.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**